src/main/java/appeng/container/implementations/ContainerMEMonitorable.java

```diff
@@ -135,6 +136,19 @@ else if( monitorable instanceof IGridHost )
 						}
 					}
 				}
+				else if( monitorable instanceof IActionHost )
+				{
+					final IGridNode node = ( (IActionHost) monitorable ).getActionableNode();
+					if( node != null )
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS134)